pingdom-checks/benefits-reference-data.ping

```diff
@@ -17,10 +17,10 @@ pingdom save-check \
   -a responsetime_threshold="30000" \
   -a resolution="1" \
   -a group="benefits-services" \
-  -a userids_csv="$NO_USERS" \
-  -a integrationids_csv="$BENEFITS_REFERENCE_DATA_SLACK_ID"
+  -a userids_csv="$STATUSPAGE_SANDBOX_BENEFITS_REFERENCE_DATA_V1" \
```

Comment:
The StatusPage userid should be integrated with the Backend Healthcheck (readiness probe) instead of the API healthcheck (liveliness).

Reply:
Swapped it over to the backend healt check